### DISCUSSION TITLE
Can I have part of my site in a differnt locale?

### DISCUSSION THREAD

**Discussion by YoniChechik** [facebook/docusaurus#8847] [General]
I don't want to translate the site, but I do wish to have a part of my site in a different RTL locale, and the rest of the site in English.

Is that possible?

**Comment by YoniChechik** [comment]
I've found out that I can put this lines in my md files and it solves what I need (Note: It should be after the h1):
```
# מחרוזות ומספרים

<head>
  <html lang="he" dir="rtl" />
</head>
```

Am I missing anything importent when doing so?

**Reply by slorber** [reply]
Not 100% sure it will work in all situations but worth giving it a try. I don't really have any other idea
